lib/TPP/DefaultTppPasses.cpp

```diff
@@ -143,11 +144,11 @@ struct DefaultTppPasses
         pm.addNestedPass<func::FuncOp>(createLoopInvariantCodeMotionPass());
         pm.addNestedPass<func::FuncOp>(createVectorizationPass());
 
-	//Please note, canonicalizer should be after hoisting pass because 
-	//it fuses outer tiling loops and it results in no pattern 
-	//matching for hoisting pass. Moved inside VectorToKernel Path.
-        
-	if (vectorToXSMM) {
+        // Please note, canonicalizer should be after hoisting pass because
+        // it fuses outer tiling loops and it results in no pattern
+        // matching for hoisting pass. Moved inside VectorToKernel Path.
```

Comment:
You can remove this comment now. It was meant for you to make sure you didn't need it.